### ISSUE TITLE
Error when accessing keybase filesystem on MacOS after waking from sleep

### ISSUE THREAD

**Issue by jamesmcdonald** [keybase/client#3541]
Just tried to cd to /keybase/public/jamesmcdonald and got a series of timeouts. Clicking on keybase in the menu bar asked for my passphrase, and then it worked again.

my log id: c04a7dbbf985a80d3e534a1c

**Comment by strib** [comment]
Yeah, from the logs it seems like it was stuck waiting for authentication.  When you say "Clicking on keybase in the menu bar", do you mean there was already an open password prompt that you just hadn't noticed before?  Or were you opening the little Keybase icon in the top status bar?

**Comment by jamesmcdonald** [comment]
I clicked the icon in the menu bar. It hadn't popped up a password prompt, though sometimes it does when waking up. It's not that bothersome, it just asked me to send the log so I thought it might be useful if you want to hone the UX.
